docs/superpowers/plans/2026-07-13-shift-trade-directed-rls-plan.md

```
@@ -0,0 +1,56 @@
+# Plan — Enforce directed-trade privacy in shift_trades RLS
+
+**Design:** docs/superpowers/specs/2026-07-13-shift-trade-directed-rls-design.md
+**Branch:** `fix/shift-trade-directed-rls`
+
+TDD via pgTAP (RED: failing visibility test against current policy → GREEN: migration).
+
+---
+
+### Task 1 — pgTAP RLS test (RED first)
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)